src/app/shared/menu/export/export-dropdown.component.spec.ts

```diff
@@ -103,9 +122,22 @@ describe('ExportDropdownComponent', () => {
     window.dispatchEvent(new Event('scroll'));
   });
 
-  it('should dispatch google analytics actions onExportCsv', () => {
+  it('should dispatch google analytics action onExportCsv', () => {
     jest.spyOn(store, 'dispatch');
     comp.onExportCsv();
     expect(store.dispatch).toHaveBeenCalledWith(new GoogleAnalyticsEventAction({gaAction: 'exportCSV'}));
   });
+
+  it('should dispatch google analytics action when finished creating Google Sheet', () => {
+    jest.spyOn(store, 'dispatch');
+    expect(store.dispatch).not.toHaveBeenCalled();
+    comp.onExportGoogleSheet();
+    expect(store.dispatch).toHaveBeenCalledWith(new GoogleAnalyticsEventAction({gaAction: 'exportGoogleSheet'}));
```

Comment:
In addition to tracking the routes a user takes through the app, GA  alsojust has an events feature that we use throughout the app, and you can send any string. (I honestly could stand to do a review of all the event strings and standardize them into a set of consts since I've been reviewing the events in GA lately for new features.)